exercises/Chapter1.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Chapter 1 Exercises"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Question 1\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are two statements that correspond to the *Probability of being sunny given that it is the 9th of July of 1816*\n",
+    "1. p(sunny | 9th of July of 1816)\n",
+    "2. p(sunny| 9th of July of 1816) / p(9th of July of 1816)\n",
+    "\n",
+    "For the second one recall the product rule (Equation 1.1)\n",
+    "\n",
+    "$$ p(A,B) = p(A|B)p(B)$$\n",
+    "\n",
+    "A rearrangement of this formula yields\n",
+    "\n",
+    "$$ p(A|B) = \\frac{p(A,B)}{p(B)}$$\n",
+    "Replace A and B with \"sunny\" and \"9th of July of 1816\" to get the equivament formulation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Question 2\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's assume there are 6 billion humans in the galaxy and there is only 1 Pope, Pope Francis, at time of writing. If a human is randomly picked the chances of that human being the pope are 1 in 6 billion. In mathematical notation\n",
+    "\n",
+    "$$ p(Pope | Human) = \\frac{1}{6,000,000} $$\n",
+    "\n",
+    "Additionally I am very confident that the Pope is human, so much so that I make this statement. *Given a pope, I am 100% certain they are human*. Written in math\n",
+    "$$ p(human | Pope) = 1$$\n",
+    "\n",
+    "\n",
+    "In animated show Futurama it turns out there are actually two popes, the human pope and a reptile space pope. Since the space pope is the same it doesn't change the human population so we still have\n",
+    "\n",
+    "$$ p(Pope | Human) = \\frac{1}{6,000,000} $$\n",
+    "\n",
+    "but, since there are two popes *The probability of a human, given a pope* is now 1/2, written as \n",
+    "\n",
+    "$$ p(human | Pope) = .5$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Question 3\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The priors in thie model are\n",
+    "\n",
+    "$$ \\mu \\text{~} Normal(0,10)$$\n",
+    "$$ \\sigma \\text{~} HalfNormal(25)$$\n",
```

Comment:
@aloctavodia The two priors are listed here, I can see them in my local Jupyter notebook, I'm not sure why its not rendering it Giithub. Will investigte